src/main/java/org/broadinstitute/hellbender/tools/spark/sv/discovery/alignment/ContigAlignmentsModifier.java

```diff
@@ -208,18 +208,46 @@ private static AlignmentInterval clipAlignmentInterval(final AlignmentInterval i
                 refBasesConsumed += ce.getLength();
             }
         }
+        if (newMiddleSection.size() < 2)
+            throw new GATKException("After clipping, the middle section of an alignment isn't split, which seems wrong\t" + input.toPackedString() + "\tclip length " +
```

Comment:
Can you get rid of the 'seems wrong' wording? Either it is or it isn't, so I'd just describe which assumption is being violated here.. It might be nice to put this message in terms of the state of the input alignment rather than the 'middle section' so that if a user happens to get this they can more easily debug. Ie. "The input alignment had X condition [I'm not sure what that is] that led to an invalid clipped alignment."

Reply:
improved the error message. Thanks!